Resources/Prototypes/_RMC14/Entities/Clothing/Belt/belts.yml

```diff
@@ -787,9 +787,53 @@
   - type: StorageFill
     contents:
     - id: RMCSpeedLoaderMateba
-      amount: 5
+      amount: 6
```

Comment:
This is intended to be 5
![image](https://github.com/user-attachments/assets/9c8f2158-2c62-4d12-bf32-8c138346dfce)

```suggestion
      amount: 5
```